charts/gha-runner-scale-set-controller/templates/deployment.yaml

```diff
@@ -69,10 +69,10 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         {{- with .Values.env }}
-          {{- if kindIs "slice" .Values.env }}
-        {{- toYaml .Values.env | nindent 8 }}
+          {{- if kindIs "slice" . }}
+            {{- toYaml . | nindent 8 }}
```

Comment:
do we want to kill the support for `slice`?

Reply:
I would add an L0 test for this.

Reply:
I would vote for yes, but I did not know if anyone uses it like that.
It would be a breaking change

Reply:
it never work before. 😄

Reply:
Then I'd vote for removing a slice :smile:

Reply:
I like the flexibility of retaining the slice support, less headaches for us in the future. We've already adopted this philosophy in exposing the pod spec, we might as well do it here at no cost to us